mlir/include/mlir/Dialect/XeGPU/IR/XeGPUTypes.td

```diff
@@ -157,6 +156,7 @@ def XeGPU_TensorDesc: XeGPUTypeDef<"TensorDesc", "tensor_desc",
       return MemorySpace::Global;
     }
 
+    // get the ChunkSize for blocked TensorDesc
```

Comment:
`ArrayLength`? It's better to leave `chunk` naming to scatter only

Reply:
Thanks, fixed.